addons/MagicaVoxel_Importer_with_Extensions/vox-importer.gd

```diff
@@ -91,6 +114,20 @@ func import(source_path, destination_path, options, _platforms, _gen_files):
 	else:
 		mesh = CulledMeshGenerator.new().generate(vox, voxel_data, scale, snaptoground)
 
+	# Full mesh rotation
+	if rotation_x != 0.0 or rotation_y != 0.0 or rotation_z != 0.0:
```

Comment:
So even if the rotation is 0, this code will still run fine... I think we can remove this check, and just always rotate, even if the rotation is always 0.